src/Microsoft.Extensions.Logging.Filter/Internal/FilterLoggerFactory.cs

```diff
@@ -22,12 +24,14 @@ public void AddProvider(ILoggerProvider provider)
 
         public ILogger CreateLogger(string categoryName)
         {
-            return _innerLoggerFactory.CreateLogger(categoryName);
+            throw new NotImplementedException();
```

Comment:
@lodejard I suppose this is OK as we do not want this logger factory instance to be used for anything other than creating wrapping providers?